changes/2020-06-09_wrapping-key-identifiers/change.md

```diff
@@ -0,0 +1,145 @@
+| [Encrypted Data Key](../../framework/structures.md#encrypted-data-key)                              |
+| [AWS KMS Keyring](../../framework/kms-keyring.md)                                                   |
+| [Raw AES Keyring](../../framework/raw-aes-keyring.md)                                               |
+| [Raw RSA Keyring](../../framework/raw-rsa-keyring.md)                                               |
+| [Keyring Decryption Contract](https://github.com/awslabs/aws-encryption-sdk-specification/pull/131) |
+
+## Affected Specifications
+
+This serves as a reference of all specification documents that this change affects.
+
+| Specification                                             |
+| --------------------------------------------------------- |
+| [Structures](../../framework/structures.md)               |
+| [Keyring Interface](../../framework/keyring-interface.md) |
+| [AWS KMS Keyring](../../framework/kms-keyring.md)         |
+| [Raw AES Keyring](../../framework/raw-aes-keyring.md)     |
+| [Raw RSA Keyring](../../framework/raw-rsa-keyring.md)     |
+
+## Affected Implementations
+
+The scope of this change only affects the specification.
+Follow-up changes that define
+whether implementations expose these concepts
+MAY affect implementations.
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Summary
+
+We have referenced several terms throughout various specification documents
+that are never defined:
+"key namespace",
+"key name",
+"provider ID",
+and "provider info".
+This change defines each of these terms
+and describes their relationships to each other
+and the keyrings they represent.
+
+## Out of Scope
+
+- Whether, or how, each keyring exposes any of these values at runtime is out of scope.
+
+- Specific values for these terms for any specific keyring is out of scope.
+
+## Motivation
+
+Various specification documents reference
+"key namespace",
+"key name",
+"provider ID",
+and "provider info",
+but we never define what these terms mean
+or how they relate to each other.
+
+In order to ensure the specification documents accurately describe keyring behavior,
+we need to define all terms that they use.
+
+## Drawbacks
+
+This change SHOULD NOT introduce any drawbacks.
+It is identifying and describing
+terms that already exist
+and the existing relationship between them.
+
+## Security Implications
+
+This change SHOULD NOT have any security implications.
+
+## Operational Implications
+
+This change SHOULD NOT have any operational implications.
+
+## Guide-level Explanation
+
+Key namespace and key name are configuration values that MAY determine the behavior of a keyring.
```

Comment:
There is something about this MAY that makes me a little uncomfortable.
Like a keyring that does not delegate, may not even have these values.
Below you talk about how the values SHOULD be equal.
Perhaps here is a place to express the relationship more concretely?

I understand that
the "key namespace" and the "key name" as "for the keyring"
and the "provider ID and "provider info" as "for the encrypted data key".